lua/lspconfig/server_configurations/julials.lua

```diff
@@ -1,5 +1,34 @@
 local util = require 'lspconfig.util'
 
+local function activate_env()
+  local bufnr = vim.api.nvim_get_current_buf()
+  local julials_client = util.get_active_client_by_name(bufnr, 'julials')
+  if not julials_client then
+    vim.notify(
+      'method julia/activateenvironment is not supported by any servers active on the current buffer',
+      vim.log.levels.WARN
+    )
+    return
+  end
+  local julia_project_files = { 'Project.toml', 'JuliaProject.toml' }
```

Comment:
Here we are reinventing the root_dir on line 77--which also doesn't have JuliaProject.toml, which means this command has different expectations than the `root_dir` ...? Not good.

Reply:
Yeah, we can modify the `root_dir`. This is not necessarily the same as the root directory detection because it's not detecting a project root but an environment definition